src/components/SearchByText/TextSearchBox.js

```diff
@@ -4,12 +4,12 @@ import T from 'prop-types';
 import { connect } from 'react-redux';
 import { createSelector } from 'reselect';
 import { debounce } from 'lodash-es';
+import loadable from 'higherOrder/loadable';
```

Comment:
More linting than anything, I guess this is not really useful here if the corresponding element is commented out
```suggestion
// import loadable from 'higherOrder/loadable';
```